test/PoolManagerInitialize.t.sol

```diff
@@ -163,7 +163,7 @@ contract PoolManagerInitializeTest is Test, Deployers, GasSnapshot {
         // Assumptions tested in Pool.t.sol
         sqrtPriceX96 = uint160(bound(sqrtPriceX96, TickMath.MIN_SQRT_PRICE, TickMath.MAX_SQRT_PRICE - 1));
 
-        address hookEmptyAddr = Constants.EMPTY_HOOKS;
+        address hookEmptyAddr = Constants.ALL_HOOKS;
```

Comment:
The old EMPTY_HOOKS address is different than ALL_HOOKS - just checking thats intended here? 
also change var name.. hookEmptyAddress is misleading

Reply:
ok changed the variable name. I think it was just bad naming tbh but im definitely confused. All the `EMPTY_HOOKS` uses were to create a gas test for `swap with hook` or `add liq with hook` so really "empty" is just wrong lol